tables/da-dk-g28.ctb

```diff
@@ -1761,9 +1047,443 @@ noback context _$d["nd"]!$l @1345-145
 noback context $d[]$l!$l @6
 noback context $d[]$l~ @6
 
+noback pass2 @f-235 @2357
```

Comment:
Would you mind adding some comment here explaining what these pass2 rules are all about?